app/models/suggestions_job_status.rb

```diff
@@ -0,0 +1,4 @@
+class SuggestionsJobStatus < ApplicationRecord
+  validates :done, inclusion: { in: [true, false] }
+  validates :done, exclusion: { in: [nil] }
```

Comment:
I'm bad at keeping up with Rails -- is this `validates :done, :inclusion: { in: [true,false] }, presence: true, allow_nil: false`?